app/models/member.rb

```diff
@@ -58,6 +58,20 @@ class Member < ApplicationRecord
       key: Rails.application.secrets.secret_key_for_ssn_encryption,
   )
 
+  def self.insured
+    where(insured: true).
+      where(requesting_health_insurance: true).
+      order(created_at: :asc)
+  end
+
+  def self.first_member
+    limit(1).first
+  end
+
+  def self.after(member)
+    where("created_at > ?", member.created_at)
```

Comment:
thoughts on doing this based on `id` rather than `created_at`? IRL will probably be fine, but I imagine in specs this could sometimes lead to intermittent errors

Reply:
I thought about ID's too and there's a reason why I did it this way.

Considering the fact we're holding onto personal information it is possible -- maybe likely? -- that the id columns will eventually change to be UUID's instead of integers. Now, that's entirely premature -- I know this. However - created_at and ID are analogous in terms of if/when they change. I can't remember the last time I saw a models' created_at timestamp get changed. If that remained the norm then id and created_at would serve in the same capacity.